File: March 7, 2026.js
function getElementSize(windowSize, elementVw, elementVh) {

  const size = windowSize.split(" x ");
  const vw = parseInt(elementVw);
  const vh = parseInt(elementVh);
  const resultW = size[0] * (vw / 100);
  const resultH = size[1] * (vh / 100);

  return `${resultW} x ${resultH}`;
}

File: March 7, 2026.py
def get_element_size(window_size, element_vw, element_vh):

    width, height = map(int, window_size.split(" x "))
    vw = int(element_vw[:-2])
    vh = int(element_vh[:-2])

    result_w = width * vw // 100
    result_h = height * vh // 100

    return f"{result_w} x {result_h}"